src/Tools/dotnet-monitor/DotNetHost.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    internal static partial class DotNetHost
+    {
+        private const string DotNetName = "dotnet";
+        public const string ExecutableRootName = DotNetName;
+        public const string InstallationDirectoryName = DotNetName;
```

Comment:
Is there a reason why we have three constants with the same value, or is it just for readability?

Reply:
I wanted to keep separate the installation folder name vs the executable name in case we need to change that and downstream consumers of the constants can be more deliberate about why they are using the constant that they chose.